### PR TITLE
#156 add allequal method

### PR DESCRIPTION
**Pull request by MNie**
#156 , continue of #157 
`allEqual` function.
CC: @haf

### CODE REVIEW

File: Expecto/Expect.fs [hunk 1]
```diff
@@ -284,25 +284,34 @@ let exists ( actual: 'a seq) asserter message =
...
+  seq { for i in 0..(actual |> Seq.length) - 1 do
             let isDifferent = Seq.item i actual |> asserter |> not
             if isDifferent then yield (i, sprintf "%A" (Seq.item i actual))
       }
 
-  let formatResult result =
-    result
-    |> Seq.map ( fun x -> sprintf "Element at index: %d which is equal to: %A" (fst x) (snd x))
-    |> String.concat "\n"
+let inline private formatAllEqualTo actual =
```
Comment (haf) [inline Expecto/Expect.fs]: Why inline it?

File: Expecto/Expect.fs [hunk 2]
```diff
@@ -284,25 +284,34 @@ let exists ( actual: 'a seq) asserter message =
...
+  actual
+  |> Seq.map ( fun x -> sprintf "Element at index: %d which is equal to: %A" (fst x) (snd x))
+  |> String.concat "\n"
+
+/// Expect that all elements from `actual` satisfies the given `asserter`
+let all ( actual: 'a seq) asserter message =
```
Comment (haf) [inline Expecto/Expect.fs]: Stray space after `(`

File: Expecto/Expect.fs [hunk 3]
```diff
@@ -284,25 +284,34 @@ let exists ( actual: 'a seq) asserter message =
...
+  match actual with
+  | null -> Tests.failtestf "%s. Sequence is empty" message
+  | _ ->
+    let checkResult = allEqualTo actual asserter
+    if checkResult |> Seq.isEmpty then ()
```
Comment (haf) [inline Expecto/Expect.fs]: Single-par fns should not be piped.

File: Expecto/Expect.fs (hunk 1 above)
Comment (MNie) [inline Expecto/Expect.fs]: I follow the same rule as in formatSet function. I will remove that :)

File: Expecto/Expect.fs (hunk 2 above)
Comment (MNie) [inline Expecto/Expect.fs]: fix'd

File: Expecto/Expect.fs (hunk 3 above)
Comment (MNie) [inline Expecto/Expect.fs]: fix'd

File: Expecto/Expect.fs [hunk 4]
```diff
@@ -284,25 +284,34 @@ let exists ( actual: 'a seq) asserter message =
   if exist then ()
   else Tests.failtestf "%s. There isn't any element which satisfies given assertion %A." message asserter
 
-/// Expect that all elements from `actual` satisfies the given `asserter`
-let all ( actual: 'a seq) asserter message =
-  let checkThem =
-      seq { for i in 0..(actual |> Seq.length) - 1 do
+let inline private allEqualTo actual asserter =
```
Comment (haf) [inline Expecto/Expect.fs]: Let's remove this `inline` as well

File: Expecto/Flip.Expect.fs [hunk 5]
```diff
@@ -106,6 +106,9 @@ let inline exists message asserter actual = Expecto.Expect.exists actual asserte
 /// Expect that all elements from `actual` satisfies the given `asserter`
 let inline all message asserter actual = Expecto.Expect.all actual asserter message
 
+/// Expect that all elements from `actual` are equal to `equalTo`
+let inline allEqual message equalTo actual = Expecto.Expect.allEqual actual equalTo message
```
Comment (haf) [inline Expecto/Flip.Expect.fs]: And this one

File: Expecto/Expect.fs (hunk 4 above)
Comment (MNie) [inline Expecto/Expect.fs]: fix'd

File: Expecto/Flip.Expect.fs (hunk 5 above)
Comment (MNie) [inline Expecto/Flip.Expect.fs]: fix'd

**Comment by haf** [comment]
Thank you @MNie !

**Pull request merged**